couchpotato/core/plugins/base.py

```diff
@@ -171,6 +172,10 @@ def urlopen(self, url, timeout = 30, data = None, headers = None, files = None,
         headers['Connection'] = headers.get('Connection', 'keep-alive')
         headers['Cache-Control'] = headers.get('Cache-Control', 'max-age=0')
 
+        if self.getName() is "TorrentDay":
+		cookies = self.getCookies()
+		headers['Cookie'] = cookies
+
```

Comment:
You can send headers in the main torrentday function (using the urlopen function). Putting it in base isn't the correct way.